src/backend/marsha/core/models/video.py

```diff
@@ -45,6 +45,14 @@ class Video(BaseFile):
         null=True,
         blank=True,
     )
+    live_type = models.CharField(
+        max_length=20,
+        verbose_name=_("live type"),
+        help_text=_("live type."),
+        choices=LIVE_TYPE_CHOICES,
+        null=True,
+        blank=True,
```

Comment:
Add default instead of making it nullable? You necessarily have a type of live right?

Reply:
In a first time I added a default value but if the video is not a live, this property should be null. Or do you think it's not important if there is always a value ?

Reply:
I would say it's more important to be sure there is a value when it's a live. And also I'm thinking the default should be configurable because we will want jitsi to be the default but people running marsha without jitsi will want OBS as default ? Or did we make it required to have jitsi :scream:

Reply:
The type value is decided by the user in the front application. This is why adding a default value is not really relevant. You decide to create a `raw` or a `jitsi` live.

Reply:
If `None` is a possible value for this field then you have to count this possibility in your code and in your tests regardless of what the frontend may do? So removing this possibility is better if it is not desired.

Reply:
This field can be nullable. If the video is a VOD then this field is null. Like `live_state`.

Reply:
Ok. Then I would add a CheckContraint to make sure that it is not null when the video is live : https://docs.djangoproject.com/fr/3.2/ref/models/constraints/#checkconstraint

Reply:
Thanks. In this [fixup](https://github.com/openfun/marsha/pull/1005/commits/154372e8ebf73250a8b57f51630e82d13af887f2) I also add a default value in the serializer representation. WDYT ?